src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -2272,49 +1259,146 @@ export default class Aircraft {
-            // Perform standard turns 3 deg/s or 25 deg bank, whichever
-            // requires less bank angle.
-            // Formula based on http://aviation.stackexchange.com/a/8013
-            const turn_rate = clamp(0, 1 / (this.speed / 8.883031), 0.0523598776);
-            const turn_amount = turn_rate * window.gameController.game_delta();
-            const offset = angle_offset(this.target.heading, this.heading);
-
-            if (abs(offset) < turn_amount) {
-                this.heading = this.target.heading;
-            } else if ((offset < 0 && this.target.turn === null) || this.target.turn === 'left') {
-                this.heading -= turn_amount;
-            } else if ((offset > 0 && this.target.turn === null) || this.target.turn === 'right') {
-                this.heading += turn_amount;
-            }
+        // Perform standard turns 3 deg/s or 25 deg bank, whichever
+        // requires less bank angle.
+        // Formula based on http://aviation.stackexchange.com/a/8013
+        const turn_rate = clamp(0, 1 / (this.speed / 8.883031), 0.0523598776);
+        const turn_amount = turn_rate * window.gameController.game_delta();
+        const offset = angle_offset(this.target.heading, this.heading);
+
+        if (abs(offset) < turn_amount) {
+            this.heading = this.target.heading;
+        } else if ((offset < 0 && this.target.turn === null) || this.target.turn === 'left') {
+            this.heading -= turn_amount;
+        } else if ((offset > 0 && this.target.turn === null) || this.target.turn === 'right') {
+            this.heading += turn_amount;
         }
+    }
 
-        // ALTITUDE
-        let distance = null;
-        const expedite_factor = 1.5;
+    /**
+     * This updates the Altitude for the instance of the aircraft by checking the difference between current Altitude and requested Altitude
+     * @for AircraftInstanceModel
+     * @method updateAltitudePhysics
+     */
+    updateAltitudePhysics() {
         this.trend = 0;
 
         if (this.target.altitude < this.altitude - 0.02) {
-            distance = -this.model.rate.descent / 60 * window.gameController.game_delta();
+            this.decreaseAircraftAltitude();
+        } else if (this.target.altitude > this.altitude + 0.02) {
+            this.increaseAircraftAltitude();
+        }
+    }
 
-            if (this.mode === FLIGHT_MODES.LANDING) {
-                distance *= 3;
+    /**
+    * Decreases the aircrafts altitude
+    * @for AircraftInstanceModel
+    * @method decreaseAircraftAltitude
+    */
+    decreaseAircraftAltitude() {
+        let distance = -this.model.rate.descent / 60 * window.gameController.game_delta();
+        const expedite_factor = 1.5;
+
+        if (this.mode === FLIGHT_MODES.LANDING) {
+            distance *= 3;
+        }
+
+        this.trend -= 1;
+        //TODO: This might be able to become its own function since it is repeated again in the increaseAircraftAltitude()
+        if (distance) {
```

Comment:
is there a case where `!distance` is possible?  I would advocate for an early return is thats the case to save you from this nested if block.